internal/resolution/variablesources/installed_package.go

```diff
@@ -47,7 +47,7 @@ func MakeInstalledPackageVariables(
 		// find corresponding bundle for the installed content
 		resultSet := catalogfilter.Filter(allBundles, catalogfilter.WithBundleImage(bundleImage))
 		if len(resultSet) == 0 {
-			return nil, fmt.Errorf("bundleImage %q not found", bundleImage)
+			return nil, fmt.Errorf("bundle with image %q not found in available catalogs but is currently installed via BundleDeployment %q", bundleImage, bundleDeployment.Name)
```

Comment:
@everettraven this is basically what you suggested [here](https://github.com/operator-framework/operator-controller/pull/499#discussion_r1382190439) but with a slighlty modified message.

I'm trying to make an emphasis on the fact that the bundle is not fund in the _currently available_ catalogs.

Let me know what you think.

P.S.: this PR will need a rebase after we merge #499.